cpg-language-python/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/python/StatementHandler.kt

```diff
@@ -1116,4 +1104,18 @@ class StatementHandler(frontend: PythonLanguageFrontend) :
         declStmt.addDeclaration(decl)
         return declStmt
     }
+
+    /**
+     * Checks whether [mightBeAsync] has the [IsAsync] property and adds a warning to the
+     * corresponding [parentNode] stored in [Node.additionalProblems].
+     */
+    private fun addAsyncWarning(mightBeAsync: Python.AST.AsyncOrNot, parentNode: Node) {
+        if (mightBeAsync is IsAsync) {
+            parentNode.additionalProblems +=
+                newProblemDeclaration(
```

Comment:
Should this be of type `Parsing` or `Translation`? We can parse the stuff but we can't model it adequately, so I think it should be `Translation`.

Reply:
done